.github/workflows/ci.yml

```diff
@@ -20,7 +20,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        env: ["isort", "docs", "py36-django22"]
+        python: ['3.6', '3.7', '3.8' ]
+        django: [ '2.2', '3.2' ]
```

Comment:
good improvement, but we lost tox envs `isort` and `docs` here.

can you:

*  add a separate docs workflow job like in Open Zaak: https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/ci.yml#L180 and 
* add a code_quality.yml workflow that runs at least isort, you can use https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/code_quality.yml for inspiration